multisrc/overrides/wpmangareader/gabutscans/src/GabutScans.kt

```diff
@@ -10,4 +15,33 @@ class GabutScans : WPMangaReader(
 ) {
 
     override val hasProjectPage = true
+
+    override fun searchMangaFromElement(element: Element) =
+        super.searchMangaFromElement(element).apply {
+            thumbnail_url = element.select("img").attr("abs:data-src")
+        }
+
+    override fun mangaDetailsParse(document: Document) = super.mangaDetailsParse(document).apply {
+        thumbnail_url = document.select(seriesThumbnailSelector).attr("abs:data-src")
+    }
+
+    override fun pageListParse(document: Document): List<Page> {
+        val pages = mutableListOf<Page>()
+
+        // Prefer using sources loaded from javascript
+        // because current CDN (Statically) can't load old images.
+        // Example: https://gabutscans.com/cinderella-wa-sagasanai-chapter-41-end-bahasa-indonesia/
+
+        val docString = document.toString()
+        val imageListRegex = Regex("\\\"images.*?:.*?(\\[.*?\\])")
```

Comment:
```suggestion
        val imageListRegex = Regex("""\"images.*?:.*?(\[.*?])""")
```
Better use [raw strings](https://kotlinlang.org/docs/basic-types.html#string-literals). `]` is enought for the right bracket - you will be prompted about this if you use Android Studio.

Reply:
I just copied it from the parent class 😅 but it's OK 👍